src/js/adapter-loader.js

```diff
@@ -0,0 +1,61 @@
+(function() {
+  var adapterURL = localStorage.alternativeAdapterVersion
+    ? 'https://cdnjs.cloudflare.com/ajax/libs/webrtc-adapter/' + localStorage.alternativeAdapterVersion + '/adapter.min.js'

```

Comment:
Could you try to send a new commit and see if it triggers travis? You start by changing all the lines to be max 80 chars long (the travis ci will complain about this anyway when it actually runs).